api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
Did we have to change outLen to a pointer? The way we were doing this before looks fine. If so, let's create a v4 branch and bump the version. That way anyone using v3 won't break because of the api changing.